### DISCUSSION TITLE
`git log` does not seem to show respect the colors of `git diff` (using delta)

### DISCUSSION THREAD

**Discussion by bytebeast** [dandavison/delta#1700] [Q&A]
being a bit nitpicky , but...

i'm using catppucino color theme palette. `git log` seems to use the default colors in the boxed header and not respect the colors in the bat color scheme, even though its clearly using `bat` as part of `delta`. only the header has colors. is there any way it could inherit the same colors of the header in theme... just list `git diff` is using in header?

**Comment by dandavison** [comment]
Can you post a screenshot and the output of `delta --show-config` and `env | grep THEME`?

>  its clearly using bat as part of delta

What do you mean here exactly? delta doesn't call bat, but it does share the same library of color themes.
